app/js/reducers/labOrders/labOrderableReducer.js

```diff
@@ -0,0 +1,24 @@
+import {
+  FETCH_LAB_ORDERABLE_SUCCESS,
+  FETCH_LAB_ORDERABLE_FAILURE,
+} from '../../actions/actionTypes';
+import initialState from '../initialState';
+
+export default (state = initialState.labOrderables, action) => {
+  switch (action.type) {
+    case FETCH_LAB_ORDERABLE_SUCCESS:
+      return {
+        ...state,
+        ...action.orderables,
+      };
```

Comment:
There is a little bit of conflict between the way this reducer is working and the way you have your state structured.
This is what is in the state: 
```
labOrderables: {
    error: false,
    loading: false,
  },
````
And this is what is here in the reducer: 
```
return {
        ...state,
        ...action.orderables,
      };
```
Shouldn't there be a state property to store the `orderables` themselves so that they don't mess up with other data that is in the state(`loading` and `error`)?

Reply:
Done